src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -230,7 +230,7 @@ private static SystemEnforcementMode GetAppLockerPolicy(string path, SafeHandle
 
                                 // AppLocker fails when you try to check a policy on a file
                                 // with no content. So create a scratch file and test on that.
-                                string dtAppLockerTestFileContents = AppLockerTestFileContents + DateTime.Now;
+                                string dtAppLockerTestFileContents = AppLockerTestFileContents + DateTime.UtcNow.Ticks;
```

Comment:
```suggestion
                                string dtAppLockerTestFileContents = AppLockerTestFileContents + DateTime.UtcNow.ToString("u");
```

Reply:
I took real world measurements using PerfView 😋

It seems my suggested change is not much of an improvement, I measured 8ms for the code line.

I also measured the original code line and it was 14ms, same as your measurement.

Reply:
DateTime.UtcNow is more fast - it does not read current user time zone and does not convert to local time.